test/sad.js

```diff
@@ -53,16 +53,6 @@ test('invalid feed SSB URI cannot be converted to sigil', (t) => {
   t.false(ssbUri.toFeedSigil('ssb:'));
 });
 
-test('bendybutt feed URI cannot be converted to sigil', (t) => {
-  t.plan(1);
-  t.notOk(ssbUri.toFeedSigil(fixtures.feed.uri4));
-});
-
-test('gabbygrove feed URI cannot be converted to sigil', (t) => {
-  t.plan(1);
-  t.notOk(ssbUri.toFeedSigil(fixtures.feed.uri5));
-});
-
```

Comment:
It's a decently written PR (as always!) but as these tests show, I wanted to be explicit about feed sigils being specific to "classic SSB" and not supporting alternative feeds.

You raised a good point, that ssb-keys could be updated to handle URIs, but I think we need some careful discussion before we commit to that type of approach (it may be hard to revert that decision if it turns out to not be a good decision). In the meantime, I recommend doing this kind of approach, which is a tiny bit more verbose than a 1-liner: https://github.com/ssb-ngi-pointer/ssb-meta-feeds/blob/f28e6e0bdf1518acdf1c2e93c9a8668aa85d68df/keys.js#L55-L58

Reply:
That makes sense to me. I'll implement your suggestion in ssb-meta-feeds.